tests/Qutee/Tests/WorkerTest.php

```diff
@@ -203,7 +203,13 @@ public function testCallingRunSleepsUpToDefinedInterval()
         $this->object->run();
         $end    = microtime(true) - $start;
 
-        $this->assertEquals(0.8337, $end, '', 0.001);
```

Comment:
This depends on your computer/platform to get this value. Give it a margin error bigger.

:+1: